bundler/bundles/sketch.json

```diff
@@ -0,0 +1,7 @@
+{
+	"blocks": [ "sketch" ],
+	"version": "1.0.0",
+	"name": "Sketch Block",
+	"description": "“There's nothing more difficult than a line.” — Pablo Picasso",
```

Comment:
need to update this one

Reply:
Nothing for now. Addressed in c47ddd8ddce06a76eef4d381944b144f946851b1